core/templates/external-js/core-url.tid

```diff
@@ -0,0 +1,3 @@
+title: $:/core/save/offline-external-js/coreURL
+
+tiddlywikicore-5.2.4-prerelease.js
```

Comment:
The version should not be hard coded here, but I don't know how to make that happen.

Reply:
The typical use case would be to change this default value to a url like `https://some-common-url-maybe-a-cdn.example.com/path/tiddlywikicore-5.2.4-prerelease.js`.